phunware/maas-mapping-ios-sdk
# SWMAAS-1992 Add sample for using local custom POI images

Changes:
-
* Added sample to demonstrate use of custom POI image loading.
* Create alert controller on main thread, and not just present it on the main thread.

Testing:
-
1) Update your local workspace for MapKit to use the `feature/SWMAAS-1991` branch.
2) Edit the podfile for MapScenarios to point to your local MapKit workspace.
3) Run `pod update`
4) Unzip and apply the patch contained in [this](https://github.com/phunware/maas-mapping-ios-sdk/files/4371807/custom_poi_setup.patch.zip) zip file
5) Build and launch the MapScenarios app
6) Run the "Custom POI Image" sample
7) Notice that all of the POI annotation images are a red circle. This is the placeholder image used by the sample app for all POI annotations.
